src/redux/Books/book.js

```diff
@@ -0,0 +1,21 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const bookSlice = createSlice({
+  name: 'books',
+  initialState: [],
+
+  reducers: {
+    addBook: (state, action) => {
+      const newBook = {
+        id: Date(),
+        title: action.payload.title,
+      };
+      state.push(newBook);
+    },
+    removeBook: (state, action) => state.filter((book) => book.id !== action.payload.id),
+  },
+
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/215745007-7e5191a4-2534-4c4d-b044-19aba476e67d.png)
**[Optional]**

- [ ] You can see in that image above that your commits are not capitalized :fearful: It would be nice if you capitalize them to make your pull request descriptive and professional :+1: